### ISSUE TITLE
Unit tests for core utils

### ISSUE THREAD

**Issue by Nullably** [CoinAlpha/hummingbot#3265]

**Comment by Nullably** [comment]
- hummingbot/core/utils/trading_pair_fetcher.py
- hummingbot/core/utils/asyncio_throttle.py
- hummingbot/core/utils/async_retry.py
- hummingbot/core/utils/estimate_fee.py
- hummingbot/core/utils/ethereum.py
- hummingbot/core/utils/ssl_cert.py
- hummingbot/core/utils/tracking_nonce.py
- hummingbot/core/utils/trading_pair_fetcher.py
- hummingbot/core/utils/wallet_setup.py

**Comment by james-hummingbot** [comment]
Put your name by the file you are working on. Check it once it is merged into refactor/unit_tests
